src/container/RelayRenderer.js

```diff
@@ -123,10 +124,17 @@ class RelayRenderer extends React.Component {
     super(props, context);
     this.mounted = true;
     this.state = this._runQueries(this.props);
+
+    this.props.relayContext.injectBatchingStrategy(
+      ReactDOM.unstable_batchedUpdates
+    );
```

Comment:
`ReactDOM.unstable_batchedUpdates` is not suitable for server-side rendering. And generally, I believe `RelayRenderer` should not be responsible for defining the batching strategy, and should not override it either. It may cause problems such as: https://github.com/denvned/isomorphic-relay-router/issues/5#issuecomment-156920743. Still, `ReactDOM.unstable_batchedUpdates` might be the default injected by the constructor of `RelayContext`.